css/style.css

```diff
@@ -0,0 +1,3958 @@
+  margin-left: 10px;
+}
+
+.search-input::-webkit-input-placeholder {
+  font-family: var(--font-paragraph);
+  font-weight: var(--font-normal);
+  font-size: var(--primary-font-size);
+  line-height: var(--primary-line-height);
+  color: var(--opacity-50);
+}
+
+.search-input::-moz-placeholder {
+  font-family: var(--font-paragraph);
+  font-weight: var(--font-normal);
+  font-size: var(--primary-font-size);
+  line-height: var(--primary-line-height);
+  color: var(--opacity-50);
+}
+
+.search-input:-ms-input-placeholder {
+  font-family: var(--font-paragraph);
+  font-weight: var(--font-normal);
+  font-size: var(--primary-font-size);
+  line-height: var(--primary-line-height);
+  color: var(--opacity-50);
+}
+
+.search-input::-ms-input-placeholder {
+  font-family: var(--font-paragraph);
+  font-weight: var(--font-normal);
+  font-size: var(--primary-font-size);
+  line-height: var(--primary-line-height);
+  color: var(--opacity-50);
+}
+
+.search-input::placeholder {
+  font-family: var(--font-paragraph);
+  font-weight: var(--font-normal);
+  font-size: var(--primary-font-size);
+  line-height: var(--primary-line-height);
+  color: var(--opacity-50);
+}
+
+.btn-search {
+  padding: 2px;
+  display: block;
+  border: none;
+  background-color: transparent;
+  position: absolute;
+  right: 20px;
+}
+
+.btn-search:hover .icon-search,
+.btn-search:active .icon-search,
+.btn-search:focus .icon-search {
+  fill: var(--state-change-color);
+}
+
+.icon-search {
+  fill: var(--accent-color);
+  vertical-align: middle;
+  -webkit-transition: fill var(--trasition-effect);
+  -o-transition: fill var(--trasition-effect);
+  transition: fill var(--trasition-effect);
+}
+
+.btn-cart {
+  text-align: center;
+  padding: 5px 10px;
+  background-color: var(--bg-color);
+  border-radius: 50%;
+  -webkit-transition: background var(--trasition-effect);
+  -o-transition: background var(--trasition-effect);
+  transition: background var(--trasition-effect);
+}
+
+.cart-icon-header {
+  fill: var(--accent-color);
+  vertical-align: text-top;
+  -webkit-transition: fill var(--trasition-effect);
+  -o-transition: fill var(--trasition-effect);
+  transition: fill var(--trasition-effect);
+}
+
+.btn-cart:hover,
+.btn-cart:active,
+.btn-cart:focus {
+  background-color: var(--accent-color);
+}
+
+.btn-cart:hover .cart-icon-header,
+.btn-cart:active .cart-icon-header,
+.btn-cart:focus .cart-icon-header {
+  fill: var(--secondary-color);
+}
+
+/**************** Main **************************/
+
+.main {
+  display: -ms-grid;
+  display: grid;
+  row-gap: 120px;
+}
+
+/************** Hero section *********************/
+
+.hero {
+  background-image: -webkit-gradient(
+    linear,
+    left top,
+    right top,
+    color-stop(44%, #f0f6fd),
+    to(#ffffff)
+  );
+  background-image: -o-linear-gradient(left, #f0f6fd 44%, #ffffff 100%);
+  background-image: linear-gradient(90deg, #f0f6fd 44%, #ffffff 100%);
+  background-color: var(--secondary-color);
```

Comment:
Excessive style - the background is already white

Reply:
#Fixed